docs/source/api.rst

```diff
@@ -1,10 +1,28 @@
+=================
 API Documentation
 =================
 
-This documentation details the modules, classes, and functions in ``mdaencore``.
+This section contains an overview of the ``mdaencore`` API.
+For higher level examples of how to use the software, see the `ENCORE section <https://userguide.mdanalysis.org/stable/examples/analysis/trajectory_similarity/clustering_ensemble_similarity.html>`_ of the MDAnalysis User Guide.
+Note that the user guide imports the ENCORE implementation in the core library, as the current version of MDAnalysis still has this functionality.
+To use these examples, import ``mdaencore`` rather than ``MDAnalysis.analysis.encore``.
```

Comment:
Make these 3 lines a `.. see also::` block on the landing page. The API docs can be terser, the landing page should have all the "how do I" / "what does this look like in practice" information.

Reply:
Moved!